### PR TITLE
[TEP Process] TEPs may be merged offline

### PR DESCRIPTION
**Pull request by lbernick**
This commit updates the TEP process description to state that
after approval requirements have been met, no additional confirmation
is required to merge the TEP, and merging does not need to occur
during any meeting.

### CODE REVIEW

File: teps/README.md
Comment (lbernick) [reply teps/README.md:145]: No, I was referring to the person who wrote the TEP. My rationale here is that we use LGTM more for merging than for actual review; for example, the person facilitating API WG can LGTM TEP PRs without actually having reviewed them-- so why prevent the TEP author from doing so, as long as the TEP has the necessary approvals? Or are you saying this is actually prevented by automation? Happy to change this if you think it's not appropriate.

Comment (lbernick) [reply teps/README.md:145]: ah ok, I'll change this then, thanks!

**Comment by jerop** [comment]
/approve

hold until other governing board members approve changes to TEP processes
/hold 

cc @priyawadhwa @vdemeester @afrittoli @dibyom

**Comment by jerop** [comment]
@afrittoli @priyawadhwa please take a look

**Review by dibyom — APPROVED** [review]

File: teps/README.md
Comment (pritidesai) [inline teps/README.md:145]: the guideline here `or sooner by anyone with the permissions` facilitated merging TEPs offline ...

File: teps/README.md
Comment (pritidesai) [inline teps/README.md:151]: Should we still continue to review TEPs in the [API WG](https://docs.google.com/document/d/17PodAxG8hV351fBhSu7Y_OIPhGTVgj6OJ2lPphYYRpU/edit#heading=h.3zh9uuzdo77w) since its no longer part of the guidelines?

File: teps/README.md
Comment (lbernick) [reply teps/README.md:145]: I think it did in principle, but in practice, we encouraged people to check in with each other on slack if merging TEPs outside of API WG, even if nothing had changed. That's what I'm aiming to improve here

Comment (pritidesai) [reply teps/README.md:145]: Posting on a TEP slack channel is not a requirement, its just an option if the author or reviewers would like to chat in a group setting. The author and/or reviewers are more then welcome to chat via github comments or one-on-one or via any other means of communication.

I suggest keeping the original reference to the API WG as is:

```
Once all assigned reviewers
have approved the PR, the final [`/lgtm`](../process.md#prow-commands) can be added in
[the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api) or sooner by anyone with permission if needed.
```

And now, we can elaborate on the **sooner**  or **offline** option. 

The reason I am requesting to keep API WG as the first option is creating a TEP is just another PR, and a PR just needs an approval and lgtm so that it can be merged. We do not have any automation to restrict and check if all the reviewers have approved the TEP. We have seen a couple of TEPs where just one reviewer had approved it and a collaborator (outside of list of reviewers) added `lgtm` which resulted in merging of those TEPs.

**Review by priyawadhwa — APPROVED** [review]

**Comment by lbernick** [comment]
@pritidesai @jerop I've addressed the feedback from the discussion at today's governing board meeting. In particular, I have explained that we review TEPs at working group meetings, but that there are multiple channels for finding reviewers for TEPs and discussing them, and that a TEP may be merged once it has met our approval requirements. I have also clarified who are the "people with permissions" to perform these prow commands by linking to the appropriate sections of the contributor ladder.

@tektoncd/governing-board PTAL

**Comment by lbernick** [comment]
@bendory I'm going to remove the TEP label since we typically use this for reviewing TEPs, not reviewing changes to the TEP process

/test pull-community-teps-lint

File: teps/README.md
Comment (pritidesai) [inline teps/README.md:156]: Thanks @lbernick for the updates, appreciate your patience 🙏 

Instead of `but there's no requirement for TEPs to be assigned to reviewers, discussed, or merged during this time.`

We can add an additional bullet with a separate paragraph:

```
The TEP author can find required reviewers as soon as the PR is created. Following are the other great ways to find reviewers and publicize your proposed changes:

- Reaching out to the stakeholders or any community maintainers through the PR comments.
- The "tep" channel on
[slack](https://github.com/tektoncd/community/blob/main/contact.md#slack)
- The [tekton-dev mailing list](https://github.com/tektoncd/community/blob/main/contact.md#mailing-list)
```

File: teps/README.md
Comment (pritidesai) [reply teps/README.md:166]: Originally, the expectation was to one reviewer to approve the TEP PR and another reviewer to add `lgtm`. This caused a conflict where some other stakeholder (bug reporter or feature requester) raising an objection or requesting more details were not addressed before merging the PRs. It also caused delays since `lgtm` label is removed on every change. These were the reasons we adopted a new process where all assigned reviewers are expected to `approve` and the TEP author can request merging the PR after all reviewers have signed off.

**Comment by lbernick** [comment]
thanks @pritidesai I've made the suggested changes!

**Comment by pritidesai** [comment]
thanks @lbernick 👍 

/lgtm

**Comment by lbernick** [comment]
thanks Priti!

/hold cancel

**Pull request merged**
